3723/1/taker.rb

```diff
@@ -0,0 +1,39 @@
+require 'nokogiri'
+
+module Taker
```

Comment:
Module name is not clear at all, is there more precise name options?

Reply:
Done